Code/src/main/java/nl/utwente/group10/ui/components/OutputBlock.java

```diff
@@ -0,0 +1,67 @@
+package nl.utwente.group10.ui.components;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import nl.utwente.ewi.caes.tactilefx.fxml.TactileBuilderFactory;
+
+import java.io.IOException;
+
+/**
+ * OutputBlock is an extension of Block that only provides a display of the input
+ * it receives through it's inputAnchor.
+ * The input will be rendered visually on the Block.
+ * OutputBlock can be empty and contain no value at all, the value can be altered at any time
+ * by providing a different input source using a Connection.
+ */
+public class OutputBlock extends Block {
+	
+	/** The input this Block is receiving.**/
+	private String input;
```

Comment:
Can be implemented when ui-connections has been completed. Blocks will then be able to pull values from Anchor objects.